test/babelish/test_android2csv.rb

```diff
@@ -21,15 +21,39 @@ def test_initialize
       :csv_filename => csv_filename,
       :headers => headers,
       :filenames => filenames
-      })
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.